pb/rpc.proto

```diff
@@ -12,6 +12,7 @@ message RPC {
 	}
 
 	optional ControlMessage control = 3;
+    optional bytes extData = 4; // protocol extension data
```

Comment:
gofmt.

Reply:
https://developers.google.com/protocol-buffers/docs/proto3#any?

Reply:
oops! emacs did this.

Reply:
```el
(use-package go-mode
  :config (gofmt-before-save))
```

:wink:

Reply:
Any is probably a bad idea because it needs metadata, which may not be supported by some programming languages.
Also, our protobuf is proto2 so we don't want to pollute with proto3 constructs.

Reply:
I do have a gofmt save hook, but this is a protobuf file!

Reply:
:roll:

So, I'm just cheating and using spacemacs. `(add-to-list 'protobuf-mode-hooks (lambda () (setq-local indent-tabs-mode t))`?